ui/app/src/shared-worker.ts

```diff
@@ -162,6 +164,7 @@ function broadcast(message: StandardAction, excludedClient?: MessagePort) {
 }
 
 function openSocket({ site, xsrfToken }) {
+  let socketClient = site ? siteSocketClient : rootSocketClient;
```

Comment:
Make sure this wasn't called without a `site` before and it will start connecting/disconnecting to the new topic based on this change